src/server/controllers/customer.controller.js

```diff
@@ -1,13 +1,180 @@
+import jwt from 'jsonwebtoken'
+import dotenv from 'dotenv'
+
+import CustomerModel from "../models/customer.model.js"
+import { userJsonReponse, userCreateUpdateJson } from '../helper/index.js'
+
+dotenv.config()
+
 const getCustomers = (req, res) => {
-    return res.status(200).json({
-        statusCode: 200,
-        message: {
-            data: "Hello, I am your customer!"
+    CustomerModel.find({}, (err, data) => {
+        if (err) {
+            console.log(err)
+            res.status(200).json({
+                statusCode: 500,
+                error: err
+            })
+            return
+        }
+        console.log(data)
+        res.status(200).json({
+            statusCode: 200,
+            data: data?.map(d => userJsonReponse(d)) || []
+        })
+    })
+}
+
+const getCustomerByID = (req, res) => {
+    const id = req.params?.id
+    if (!id) {
+        res.status(200).json({
+            statusCode: 401,
+            error: { message: 'Invalid User Id' }
```

Comment:
Also maybe an `enum` or object to messages

Reply:
We can have an object where kwy could be the status code, and maybe another key for message